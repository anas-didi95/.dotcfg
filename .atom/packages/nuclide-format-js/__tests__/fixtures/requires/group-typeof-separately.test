const Z = require('Z');
import typeof Da from 'Da';
import typeof Bla from 'Bla';
import type A from 'A';

const x: A<Bla, Da> = Z;
