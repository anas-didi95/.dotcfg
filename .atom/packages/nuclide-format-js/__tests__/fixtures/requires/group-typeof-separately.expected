import type A from 'A';

import typeof Bla from 'Bla';
import typeof Da from 'Da';

const Z = require('Z');

const x: A<Bla, Da> = Z;
